src/utils/response/responseHandlers.ts

```diff
@@ -52,10 +52,18 @@ export async function toApiResponse<ResourceName extends ResourceNames>(
     return apiResponse;
   }
 
+  let errorMessage = 'Something went wrong';
+
+  if (typeof apiResponse.ErrorExtended?.errorString == 'string' && 0 < apiResponse.ErrorExtended.errorString.length) {
+    errorMessage = apiResponse.ErrorExtended.errorString;
+  } else if (apiResponse.ErrorMessage) {
+    errorMessage = apiResponse.ErrorMessage;
+  }
+
   throw {
     response: apiResponse,
     _plainError: resolved,
-    message: (apiResponse.ErrorMessage as string | undefined) ?? (resolved.status != 200 ? resolved.statusText : 'Unsuccessful response'),
+    message: errorMessage,
```

Comment:
@whosshell this is a breaking change, note it in the RN and make a v1.2.0

Reply:
Will do